### PR TITLE
Add object samples

### PR DESCRIPTION
**Pull request by simplicbe**

### CODE REVIEW

**Comment by simplicbe** [comment]
@froeh010 this are the most important types. One is still missing

File: sample/application/applicatin_sample.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "<guid>",
+  "name": "<localization-key>",
+  "type": "<type>", // grid, grid_structure, browser, python, clr
+  // Depending of the type, we have different settings
+  "settings": {
```
Comment (ghost) [inline sample/application/applicatin_sample.json]:
```suggestion
"configuration": {
```

File: sample/report/report_sample.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "id": "<unique-id>",
+  "name": "<name>",
+  "type": "<type>", // sql, key-value, parameter
+  "reportFile": "/private/...",
+  "configuration": {
+    "connection": "connection",
+    "provider": "<ado.net provider>",
+
+    "isListBaed": false, // only for key-value
+    "parameter": [ // only for key-value
```
Comment (ghost) [inline sample/report/report_sample.json]: is this supposed to be // only for parameter?

Comment (simplicbe) [reply sample/report/report_sample.json]: the "parameter" field is important for parameter-based and key-value reports
but the "parameter"-property looks a little bit different.

File: sample/stack_autoconnector/stack_autoconnector_sample.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "<name>",
+  "type": "xml-sql",
+  "stackId": "<stack-id>",
+  "target": "<target-stack-name>",
+  "configuration": {
```
Comment (ghost) [inline sample/stack_autoconnector/stack_autoconnector_sample.json]:
```
"configuration": {
```
are there multiple possibilities here aswell?

File: sample/stack/stack_sample.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// We will create a new service in simplic-framework for this. This service will be used in the package-system than
+{
```
Comment (simplicbe) [inline sample/stack/stack_sample.json]: Add id

File: sample/application/applicatin_sample.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "<guid>",
+  "name": "<localization-key>",
+  "type": "<type>", // grid, grid_structure, browser, python, clr
+  // Depending of the type, we have different settings
+"configuration": {
+
+  }
```
Comment (simplicbe) [inline sample/application/applicatin_sample.json]: **Type: clr:**

```json
{
    "namespace": "...",
    "class" : "...",
    "method": "..."
}
```

**Type: python:**

```json
{
    "pathh": "...",
    "class" : "...",
    "method": "..."
}
```

**Type: grid:**

```json
{
    "grid": "...",
    "connection" : "...",
    "loadOnOpen": true,
     "refreshOnSelect": false
}
```

**Type: browser:**

```json
{
    "url": "...",
    "tab": "...."
}
```

**Type: grid-structure:**

```json
{
    "stacks": [
        {
            "id": "<unique-id>",
            "stackId": "<stack-id>",
            "isVisible": true,
            "displayName": "",
            "grid": "",
            "orderId": 0,
            "registers": [
                {
                    "id": "<unique-id>",
                    "registerId": "<register-id>",
                    "displayName": "",
                    "grid": "",
                    "orderId": 0
                }
            ]
        }
    ]
}
```

Comment (simplicbe) [reply sample/application/applicatin_sample.json]: @froeh010  take a look at the possible configurations above.

File: sample/report/report_sample.json (hunk 2 above)
Comment (simplicbe) [reply sample/report/report_sample.json]: Key-Value-Report: https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/Configuration/KeyValueConfigurationModel.cs#L152
Parameter-Report: https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/Configuration/ParameterConfigurationModel.cs#L266 and https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/Configuration/ParameterConfigurationModel.cs#L278

Comment (simplicbe) [reply sample/report/report_sample.json]: Manager: https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/ReportManager.cs

File: sample/application/applicatin_sample.json (hunk 5 above)
Comment (ghost) [reply sample/application/applicatin_sample.json]: Can I find the enum to the types anywhere or is it yet to be created?

Comment (simplicbe) [reply sample/application/applicatin_sample.json]: I would represent this as string

File: sample/epl_report/epl_report.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "<unique-id>",
```
Comment (ghost) [inline sample/epl_report/epl_report.json]: internalname

File: sample/application/applicatin_sample.json (hunk 5 above)
Comment (ghost) [inline sample/application/applicatin_sample.json]: Have to add to application jsons atleast:
```
"directjump": "",
"ribbongroupid": ""
```

Comment (froeh010) [reply sample/application/applicatin_sample.json:8]: grid needs searchname property

File: sample/itembox/itembox_sample.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  "title": "<localized-name/title>",
+  "description": "<localized-description>",
+  "profiles": [
+    {
+      "displayName": "<localized-profile-name>",
+      "type": "grid",
+      "grid": "<grid-name>",
+      "isActive": true,
+      "defaultSearchString": "%",
+      "regex": null
```
Comment (ghost) [inline sample/itembox/itembox_sample.json]: this needs select statement in current table

File: sample/itembox/itembox_sample.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    }
+  ]
+}
```
Comment (ghost) [inline sample/itembox/itembox_sample.json]: itembox ident has autoincrement as of now, replace with guid?

File: sample/itembox/itembox_sample.json (hunk 7 above)
Comment (ghost) [reply sample/itembox/itembox_sample.json]: are itemboxes only used with grids? if so ignore the above.

File: sample/legacy/format_list/format_list_sample.json [hunk 9]
```diff
@@ -0,0 +1,11 @@
+// Target-Table: ESS_MS_Controls_FormatList
+{
+  "internalName": "<internal-name>",
+  "description": "<description>",
+  "items": [
+    {
+      "name": "<entry-name>",
```
Comment (ghost) [inline sample/legacy/format_list/format_list_sample.json]: is this needed? its not in the ess_ms_controls_formatlist table

File: sample/legacy/format_list/format_list_sample.json [hunk 10]
```diff
@@ -0,0 +1,11 @@
+// Target-Table: ESS_MS_Controls_FormatList
+{
+  "internalName": "<internal-name>",
+  "description": "<description>",
```
Comment (ghost) [inline sample/legacy/format_list/format_list_sample.json]: also needs displayname

File: sample/legacy/format_list/format_list_sample.json (hunk 9 above)
Comment (simplicbe) [reply sample/legacy/format_list/format_list_sample.json]: `name` --> `displayName`

File: sample/itembox/itembox_sample.json (hunk 7 above)
Comment (simplicbe) [reply sample/itembox/itembox_sample.json]: Yes, this is the Grid-Name now

File: sample/itembox/itembox_sample.json (hunk 8 above)
Comment (simplicbe) [reply sample/itembox/itembox_sample.json]: Detect update case by "SelectStatement"

File: sample/stack_context_area/stack_context_area.json [hunk 11]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (ghost) [inline sample/stack_context_area/stack_context_area.json]: probably needs id

Comment (simplicbe) [reply sample/stack_context_area/stack_context_area.json]: @froeh010 yes, Id is missing here

**Pull request merged**